core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -27,24 +27,20 @@ import com.iteneum.designsystem.components.phonenumbertext.PhoneNumberTransforma
  *
  * @author Jose G. Rivera
  * @modifiedBy Jesus Lopez
+ * @modifiedBy Irving Gonzalez
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun LpOutlinedTextFieldPassword(
     modifier: Modifier,
-    value: String,
+    value: String = "",
     isValid: Boolean,
     supportTextError: String,
     onPasswordChanged: (String) -> Unit
 ) {
-    var passwordFieldValue by remember { mutableStateOf(TextFieldValue("")) }
-
     OutlinedTextField(
-        value = passwordFieldValue,
-        onValueChange = {
-            passwordFieldValue = it
-            onPasswordChanged(passwordFieldValue.text)
-        },
+        value = value,
+        onValueChange = onPasswordChanged,
         modifier = modifier,
         label = {
             Text(
```

Comment:
check this text please, what are you writing in the text is going to be show in the name of the text field

Reply:
Change this part and and some extra parameters **LabelText** and modify **value** parameter:

<img width="546" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/7d12ffdb-948a-40d1-9216-3e8c7854ec57">

also implement this component in this example:

<img width="697" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/e2ca11d5-dd61-4aa4-8714-cdeb7c1d3542">

All works good:

<img width="368" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/e7a302b9-6581-4eb5-ad69-95eeed2b6412">

<img width="358" alt="image" src="https://github.com/irv205/droid_leaspert/assets/42255248/c47afefd-4355-4d5d-b439-0438955aea38">